Lib/test/regrtest.py

```diff
@@ -1369,11 +1369,14 @@ def skip_conditional_support(test_module,module_name):
 
     'java.nt':     # Expected to fail on Windows
         """
-        test_mailbox           # fails miserably and ruins other tests
-        # test_os_jy             # Locale tests fail on Cygwin (but not Windows)
```

Comment:
This `test_os_jy` is meant as a reminder that we've enabled the test, but are ignoring Cygwin, perhaps unkindly.

Reply:
I reran it on cygwin, though, and it passed.